.github/workflows/build.yml

```diff
@@ -14,7 +14,7 @@ jobs:
         uses: actions/setup-java@v2
         with:
           distribution: 'adopt'
-          java-version: '8'
+          java-version: '17'
```

Comment:
I think we have some incompatibility with our current tooling + Java 17. We might need to stay on 8. @jbarr21 to confirm -- not sure if still relevant

Reply:
We don't consume this internally, should be fine.